src/components/Example2.tsx

```
@@ -0,0 +1,371 @@
+
+    return (
+        <div className="veeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeryLongNaem">
+            ff
+            <h3>
+                <p><p><p><p><p><p><p><p><p><p><p>ff</p></p></p></p></p></p></p></p></p></p></p>
+            </h3>
+        </div>
+    )
+};
+
+function toString(number: any){
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)